Source/WebCore/platform/graphics/skia/SkiaRenderingSettings.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2024 Igalia S.L.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU Library General Public License
+ * along with this library; see the file COPYING.LIB.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301, USA.
+ */
+
+#pragma once
+
+#if USE(COORDINATED_GRAPHICS) && USE(SKIA)
+#include <wtf/Noncopyable.h>
+
+namespace WebCore {
+
+class SkiaRenderingSettings {
+    WTF_MAKE_NONCOPYABLE(SkiaRenderingSettings);
+public:
+    static bool forceUseCpuRendering();
+    static unsigned numberOfCpuPaintingThreads();
```

Comment:
I guess this doesn't need to be a class if only has static methods. I'm not sure about this in any case. It makes sense now with our current env vars model, but that's only a temporary solution while working on skia implementation. We need to add proper settings to use either cpu or gpu rendering, and the same for accelerated 2d canvas. Using force CPU makes sense in WPE, because GPU is the default, but in GTK we also want to use CPU when accelerated compositing is disabled. Since for now these are only used once and only by CompositingCoordinator, I would move them there as just static functions.

Reply:
Done.